_config.yml

```diff
@@ -119,6 +119,9 @@ searchgov:
   # this renders the results within the page instead of sending to user to search.gov
   inline: true
 
+  # this allows Search.gov to present relevant type-ahead search suggestions in your website's search box.
```

Comment:
Let's add "If you do not want to present search suggestions, set this value to false."

Reply:
Added!